packages/openneuro-app/src/scripts/utils/gtag.js

```diff
@@ -0,0 +1,3 @@
+export const pageview = (...args) => window && window.gtag(...args)
+
+export const event = (...args) => window && window.gtag(...args)
```

Comment:
These will need to be `typeof window !== undefined` to avoid throwing an error when it isn't. Or better `'gtag' in globalThis` to check global in Node and window in the browser main thread.